src/Sdk/Validation/Validator.php

```diff
@@ -0,0 +1,175 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\SdkBlueprint\Sdk\Validation;
+
```

Comment:
Please add a comment to explain that we can skip the attribute this way because we set a default `null` value to all attributes in `DataTransferObject` constructor to avoid any unwanted behaviour because of changes later.